StantUiIosLibraryDemo/StantUiIosLibraryDemo/DemoApplication/FirstHeader/FirstHeaderViewController.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  FirstHeaderViewController.swift
+//  StantUiIosLibraryDemo
+//
+//  Created by Mac Mini Novo on 29/05/19.
+//  Copyright © 2019 Stant. All rights reserved.
+//
+
+import UIKit
+import StantUiIosLibrary
+
+class FirstHeaderViewController: UIViewController, RecentItemCellDelegate {
+   
+    var indexSelected = 0
```

Comment:
Essas variáveis precisam ser públicas? Ou ter referência forte?

Reply:
Elas tem que ser públicas para serem acessadas nos testes.. 
Quanto a força das referências, vou refatorar geral.